.circleci/config.yml

```diff
@@ -172,9 +171,9 @@ jobs:
           #   It should match the name seen in the "sdkmanager --list" output
           system-image: system-images;android-29;default;x86
           #   The command to be run, while waiting for emulator startup
-          post-emulator-launch-assemble-command: make ci-build
+          post-emulator-launch-assemble-command: make assemble-phone
           #   The test command
-          test-command: make ci-functional-test
+          test-command: make functional-test
```

Comment:
Do you want to change functional-test-coverage now or later?

Reply:
Done!